edb/edgeql-parser/edgeql-parser-python/src/lib.rs

```diff
@@ -52,12 +52,17 @@ py_module_initializer!(
         m.add(
             py,
             "parse",
-            py_fn!(py, parse(parser_name: &PyString, data: PyObject)),
+            py_fn!(py, parse(spec_filepath: &PyString, parser_name: &PyString, data: PyObject)),
```

Comment:
It's a little sketchy to me that parse takes a path but will use the cached version even if it's from a different path. Maybe `parse` should take no path and always use a global cache, or maybe the cache should be keyed by the path?